composables/useGraphql.ts

```diff
@@ -28,17 +28,19 @@ export default function ({
   queryName,
   clientName = '',
   variables = {},
-  options = {},
   disabled = computed(() => false),
   data = ref(),
   error = ref(),
   loading = ref(true),
 }) {
+  const { client: clientPrefix } = usePrefix()
   const { $consola } = useNuxtApp()
-  const { prefix, client } = useQueryParams({ queryPrefix, clientName })
+
+  const prefix = queryPrefix || clientPrefix.value
+  const client = clientName || clientPrefix.value
```

Comment:
is it reactive?

Reply:
updated in 186a197f406cf18ca230cf1a9ed5a78eb1ca8d8f